base-compat-batteries/src/Prelude/Compat.hs

```diff
@@ -0,0 +1,13 @@
+{-# LANGUAGE CPP, NoImplicitPrelude, PackageImports #-}
+module Prelude.Compat (
+  module Base
+#if !(MIN_VERSION_base(4,9,0))
+, Semi.Semigroup((Semi.<>))
+#endif
```

Comment:
Would it make sense to depend and re-export `base-orphans`'s `Data.Orphans`?

Reply:
Good. Should that be documented in readme?

Reply:
Done.